src/main/java/httpserver/response/ResponseWriter.java

```diff
@@ -56,6 +55,15 @@ public Response buildSuccessResponse(Request request, String[] methods) {
             return response;
         }
 
+        if (request.path.equals("/json_response")) {
+            response = new ResponseBuilder()
+                    .setStatusCode(StatusCodes.SUCCESS)
+                    .addHeader(Constants.TYPE, "application/json;charset=utf-8")
+                    .setBody("{\"key1\":\"value1\",\"key2\":\"value2\"}")
```

Comment:
Can you try using POJOs (Plain old Java objects) that you can stringify here? That will give you more real-world practice on sending objects through REST.